Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -93,6 +93,9 @@ function InstallCUDADrivers() {
 
     ubuntu*)
         GetOSVersion
+        if [[ $os_RELEASE =~ 19.* ]]; then
```

Comment:
please add a comment that this is temporary until nvidia releases support for 19.* series